management/src/main/java/nearsoft/skytouch/management/controller/ProductController.java

```diff
@@ -0,0 +1,48 @@
+package nearsoft.skytouch.management.controller;
+
+import nearsoft.skytouch.common.model.Product;
+import nearsoft.skytouch.management.service.ProductService;
+import org.springframework.stereotype.Controller;
+import org.springframework.validation.BindingResult;
+import org.springframework.web.bind.annotation.ModelAttribute;
+import org.springframework.web.bind.annotation.RequestMapping;
+import org.springframework.web.bind.annotation.RequestMethod;
+
+import java.util.List;
+import java.util.Map;
+
+
+@Controller
+class ProductController {
```

Comment:
Shouldn't you be calling productService somewhere to add a product?

Reply:
Yes, but it is not implemented yet because the microservice module is not available yet

Reply:
Remember SOLID principles, ProductController shouldn't wait until the microservice is ready to be ready too ... that's not a responsibility of the controller

Reply:
Ok i've already add the store method in the services also y called it in the controller